addons/auth_totp/views/res_users_views.xml

```diff
@@ -18,45 +18,43 @@
-                                <div invisible="totp_enabled">
-                                    <button invisible="id == uid" name="action_totp_enable_wizard"
-                                            disabled="1" type="object" class="fa fa-toggle-off o_auth_2fa_btn disabled" aria-label="Enable 2FA"></button>
-                                            <button invisible="id != uid" name="action_totp_enable_wizard"
-                                            type="object" class="fa fa-toggle-off o_auth_2fa_btn disabled pe-auto" aria-label="Enable 2FA"></button>
-                                </div>
-                                <button invisible="not totp_enabled" name="action_totp_disable" type="object"
-                                class="fa fa-toggle-on o_auth_2fa_btn text-primary enabled" aria-label="Disable 2FA"></button>
-                            </div>
-                            <span invisible="totp_enabled" class="text-muted">
-                                Two-factor Authentication ("2FA") is a system of double authentication.
-                                The first one is done with your password and the second one with a code you get from a dedicated mobile app.
-                                Popular ones include Authy, Google Authenticator or the Microsoft Authenticator.
-                                <widget name="documentation_link" path="/applications/general/auth/2fa.html" icon="fa-fw o_button_icon fa-info-circle" label="Learn More"/>
-                                </span>
-                                <span invisible="not totp_enabled" class="text-muted">This account is protected!</span>
-                            </div>
-                        <group name="auth_devices" string="Trusted Devices" invisible="not totp_trusted_device_ids">
-                            <div colspan="2">
-                                <field name="totp_trusted_device_ids" nolabel="1" colspan="4" readonly="1">
-                                    <tree create="false" delete="false">
-                                        <field name="name" string="Device"/>
-                                        <field name="create_date" string="Added On"/>
-                                        <button type="object" name="remove"
-                                                title="Revoke" icon="fa-trash"/>
-                                    </tree>
-                                </field>
-                                <button name="revoke_all_devices" string="Revoke All" type="object" class="btn btn-secondary"
-                                        confirm="Are you sure? The user may be asked to enter two-factor codes again on those devices"/>
-                            </div>
-                        </group>
-                </page>
-            </xpath>
+            <page string="Account Security" name="account_security">
```

Comment:
```suggestion
            <page name="account_security" position="inside">
```
This is very unclear what his is doing as there is no `position="..."`
Is this position inside by default ?
Besides there is no need to repeat the `string`, for the inheritance mecanism to work.